.github/dependabot.yaml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
Periodically checks for github action updates - are we interested in this?

Reply:
I'm willing to try it. But I'm in favor of removing it the moment it starts producing noise.

Reply:
In my experience it shouldn't be too noisy as we are not using very fast changing github actions